settings.py

```diff
@@ -57,6 +57,7 @@
     ('fr', 'French'),
     ('fra', 'French'),  # we need this alias
     ('hin', 'Hindi'),
+    ('sw', 'Swahili'),
```

Comment:
This was the autocomplete for my local machine in my settings. Tested out a translation locally to make sure it worked when sw was selected

![menu_010](https://cloud.githubusercontent.com/assets/1471773/6427039/f9a56802-bf41-11e4-81e0-bb4a009d781d.png)